projects/ngx-piwik-pro/src/lib/interfaces/piwik-pro-utils.interface.ts

```diff
@@ -0,0 +1 @@
+export type LimitedArrayFiveStrings<T extends string[] = []> = [string, ...T] | [string, string, string, string, string];
```

Comment:
This isn't working. I checked it in TS playground and I cannot create an array with two or three elements.

Reply:
Fixed